archive_converter_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from archive_converter import generate_email_from_file
+
+class Archive_Converter_Test(unittest.TestCase):
+
+    def test_generate_email_from_single_email_thread(self):
+        email = generate_email_from_file("test_data/patch6.txt")
+        self.assertEqual(email.subject, '[PATCH v2 1/2] Input: i8042 - Prevent intermixing i8042 commands')
+        self.assertEqual(email.in_reply_to, None)
+        self.assertEqual(email.from_, "Raul E Rangel <rrangel@chromium.org>")
+        self.assertTrue(len(email.content) > 0)
+        
```

Comment:
can we have a test here that checks we map the parent to the right child?

E.g. calls `update()` and then we check that the subject of the child matches what we want, for example?
(Don't need it to check too deeply)